modules/ROOT/pages/understanding-orchestration-using-mule.adoc

```diff
@@ -8,11 +8,16 @@ Orchestration is how Mule facilitates the process of moving data through a Mule
 
 == Orchestration and Flows
 
-Orchestration involves decoupling point-to-point integration in favor of chains of messages sent to sources that can be reused or changed as systems or business needs change. It involves moving beyond patterns into a service-based framework, where you wrap multiple components with sources to decouple them from protocols and routing requirements, enabling multiple services to work as one when implementation requirements demand it. Some common use cases that flows easily orchestrates include simple integration, scheduled data processing, connecting cloud and on-premise applications, and event processing where multiple services must be composed and operate simultaneously.
+Orchestration involves decoupling point-to-point integration in favor of chains of operations that can be reused or changed as systems or business needs change. It involves moving beyond patterns into a service-based framework, where you wrap multiple components with sources to decouple them from protocols and routing requirements, enabling multiple services to work as one when implementation requirements demand it. Some common use cases that flows easily orchestrate include:
 
-== How Flows Enables Implementation of Orchestration
+* simple integration
+* scheduled data processing
```

Comment:
Should this say "Scheduling data processing" for consistency with the other bullets?